src/components/TodoList.tsx

```diff
@@ -0,0 +1,85 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import { Todo } from '../types/Todo';
+import { EditForm } from './EditForm';
+
+type Props = {
```

Comment:
use interface prefer. 
Types have unique features but the interface is best for scaling.